stdlib/Std/src/Time.luna

```diff
@@ -3,50 +3,165 @@ import Std.Base
 
     # The offset (in minutes) from the UTC timezone.
+    #
+    # > import Std.Time 
+    # >
+    # > def main:
+    # >    pacificStandardTime = TimeZone.pst
+    # >    min = pacificStandardTime.minutes
+    # >    None
+    #
+    # `return`: Numerical value for the offset in minutes for `self` timezone.
     def minutes: case self of
         TimeZoneVal m _ _: m
 
     # Is this timezone valid only throughout the summertime?
+    #
+    # > import Std.Time 
+    # >
+    # > def main:
+    # >    pacificStandardTime = TimeZone.pst
+    # >    justSummer = pacificStandardTime.summerOnly
+    # >    None
+    #
+    # `return`: Boolean value describing if the timezone has daylight saving time or not.
     def summerOnly: case self of
         TimeZoneVal _ s _: s
 
     # The three letter shortcut name for this timezone, can be blank.
+    #
+    # > import Std.Time 
+    # >
+    # > def main:
+    # >    pacificStandardTime = TimeZone.pst
+    # >    timeZoneName = pacificStandardTime.name
+    # >    None
+    #
+    # `return`: Text value contains the three letter shortcut name for this timezone, can be blank.
     def name: case self of
         TimeZoneVal _ _ n: n
 
     # Creates a nameless timezone with a given offset (in minutes).
+    #
+    # > import Std.Time 
+    # >
+    # > def main:
+    # >    newTimeZone = TimeZone.fromMinutes 360
+    # >    None
+    #
+    # `m`: The offset for new timezone in minutes.
+    #
+    # `return`: New nameless timezone instance with the offset `m` in minutes.
     def fromMinutes m: TimeZoneVal m False ""
 
     # Creates a nameless timezone with a given offset (in hours).
+    #
+    # > import Std.Time 
+    # >
+    # > def main:
+    # >    newTimeZone = TimeZone.fromHours 4
+    # >    None
+    #
+    # `h`: The offset for new timezone in hours.
+    #
+    # `return`: New nameless timezone instance with the offset `h` in hours.
     def fromHours h: TimeZoneVal (h * 60) False ""
 
     # Returns the `TimeZone` object representing the current timezone.
+    #
+    # > import Std.Time 
+    # >
+    # > def main:
+    # >    currentTimeZone = TimeZone.current
+    # >    None
+    #
+    # `return`: The `TimeZone` object representing the current timezone.
     def current: primGetCurrentTimeZone
 
+    # Returns the `TimeZone` object representing the UTC timezone.
+    #
+    # > import Std.Time 
+    # >
+    # > def main:
+    # >    utcTimeZone = TimeZone.utc
+    # >    None
+    #
+    # `return`: The `TimeZone` object representing the UTC timezone.
     def utc: TimeZoneVal 0 False "UTC"
 
+    # Returns the `TimeZone` object representing the GMT timezone.
+    #
+    # > import Std.Time 
+    # >
+    # > def main:
+    # >    gmtTimeZone = TimeZone.gmt
+    # >    None
+    #
+    # `return`: The `TimeZone` object representing the GMT timezone.
     def gmt: TimeZoneVal 0 False "GMT"
```

Comment:
It seems really strange to me that we have constants for some time-zones and not others. `UTC` I can understand, but why do these others get special treatment?

Reply:
I have no idea, probably this was just POC. I can add more time-zones but I think it should go in separate PR.

Reply:
Oh definitely if we want to add more.